pkg/kubernetes/scripts/virtual-machines/utils.jsx

```diff
@@ -26,3 +26,12 @@ export function getPairs(object) {
         value: object[key]
     }))
 }
+
+export function mouseClick(fun) {
+    return function(event) {
+        if (!event || event.button !== 0)
+            return;
+        event.stopPropagation();
```

Comment:
Based on https://github.com/cockpit-project/cockpit/pull/8470, this should be `preventDefault` so it will not interfere with pkg/kubernetes router if handling `<a href="#">